src/main/kotlin/org/rust/cargo/project/settings/impl/RustProjectSettingsServiceImpl.kt

```diff
@@ -42,7 +42,6 @@ class RustProjectSettingsServiceImpl(
 
     override var data: RustProjectSettingsService.Data
         get() {
-            state = state
```

Comment:
Ouch, what a blunder 😅 

I intended to write this as `val state = state`: this class should be thread safe, so it is nice to read the `state` field only once to gurantee that the return value is consistent, and not a mixture from several different states.